### DISCUSSION TITLE
Does Superface generate api

### DISCUSSION THREAD

**Discussion by atjbnkcoo** [superfaceai/.github#13] [General]
I am being told time and time again that this app will not work in my used case of connecting third-party API's for integrating with their data .

My use case is to connect with Stripe Apps?
Can Superface automate this process for me through prompts ?

**Comment by martyndavies** [comment]
Hey @atjbnkcoo, can you say more about the use case you have for connecting with Stripe Apps. More specifically what you're trying to achieve and we can let you know.

**Reply by atjbnkcoo** [reply]
Hi,
Thank you so much for reaching out.
My use case for **Superface** would be for automating the app to integrate the API's of Wise/Transferwise, Stripe API's

Not to integrate them together, but to use them separately for their own independent services.

Once integrated that data/content would need to be populated to **website** and/or **admin** ****deck****, or ****customer dashboard****

Thank you again in advance for your time. I'm looking forward to keeping the conversation moving forward and your replies.
